automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioPage.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.workbench.leonardo.rstudio
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.openqa.selenium.WebDriver
+import org.broadinstitute.dsde.workbench.leonardo.notebooks.{JupyterPage, NotebooksListPage}
+
+class RStudioPage(override val url: String)(implicit override val authToken: AuthToken,
```

Comment:
Curious what the difference is between `RStudioMainPage` and `RStudioPage` -- I thought there was only 1 rstudio page (constract with Jupyter which has the "list" page and the "notebook" page).

Reply:
I initially copied over the Notebook format because I was running into some issues running the test, but I think it was separate. Removing the extra RStudioMainPage